recipes/pyscicat/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyscicat" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rosalindfranklininstitute/pyscicat/archive/v{{ version }}.tar.gz
```

Comment:
The source URLs should all match. I'm going to assume that https://github.com/scicatproject/pyscicat is the one that is "Official" because that is the one that controls the PyPI page.

Reply:
Ok, changed